src/common/types.ts

```diff
@@ -75,8 +75,9 @@ interface JourneyStartEvent {
 interface JourneyEndEvent {
```

Comment:
If we are going to reference types on both the client and the server, it might be good to extract this types file to a top-level directory from which they both `import`, rather than having the `electron` code reference the `src` folder.

Reply:
That is a great point!! I'm wondering if this common directory was supposed to be used by both client and server? if that's the case should we just move the whole directory to a top-level?

Reply:
There is some "common" stuff that is specific to the frontend. A bunch of connector functions for interacting with the `ipc` channels that are only useful to the React code. I think it's alright to have a separate `common` directory at both levels, but if you think it's confusing we could maybe rename the frontend one to `shared` or some other thing. If we call it `shared`, let's rename the `shared.ts` to `index.ts` to avoid repeating the name. Then we could put the types in the root-level `common`.

Reply:
Thanks for clarifying it for me. I ended up creating `shared` at root directory and moved the type definition used by both electron and frontend to `types.d.ts`: https://github.com/elastic/synthetics-recorder/pull/205/commits/feeb4f5e6f4882fbf5b032f4caa99c75bf3f9eea WDYT?

Reply:
The naming convention used in Kibana and a few other places is to call it `common`. Not super important but it'd be a small change.